metron-platform/metron-solr/src/main/config/schema/error/solrconfig.xml

```diff
@@ -0,0 +1,1601 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Is this just basic stock solr? The /browse endpoint configs seem like they come from the tutorial sample, and could be mis-leading for example.